config/routes.rb

```diff
@@ -0,0 +1,18 @@
+Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  
+  # devise_for :users
+  devise_for :users, defaults: { format: :json },
+             controllers: {
+                 sessions: 'users/sessions',
+                 registrations: 'users/registrations'
+             }
+
+  resources :users 
```

Comment:
- I consider the entire CRUD for users isn't required here. According to the front-end, I don't find an option to see this. you can select the options you use here for only or except command. If you need additional help with this, consult [this article](https://stackoverflow.com/questions/6462815/except-options-in-routes-that-contain-collections-and-members-not-working).